.github/workflows/motion-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Motion Extended Tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  check-files:
+    name: Check files
+    runs-on: ubuntu-latest
+    outputs:
+      EXTENDED_MOTION_TEST: ${{ steps.file_checker.outputs.EXTENDED_MOTION_TEST }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event_name == 'pull_request_target' && github.event.pull_request.head.sha || github.ref }}
+          fetch-depth: 0
+      - name: check modified files
+        id: file_checker
+        run: |
+          ( git diff --name-only origin/${{github.base_ref}} | grep -e "motionplan" \
+            -e "referenceframe" \
+            -e "spatialmath" \
+            -e "components\/base\/wheeled" \
+            -e "components\/base\/kinematicbase" \
+            -e "services/motion/builtin" && echo "EXTENDED_MOTION_TEST=true" >> "$GITHUB_OUTPUT" ) || \
```

Comment:
Do you need to backslash escape the slashes on this line?

Reply:
I don't think so; I was testing this and forgot to remove